### DISCUSSION TITLE
Speaker adaptation/voice cloning TTS vs. Voice conversion

### DISCUSSION THREAD

**Discussion by vishal16babu** [espnet/espnet#3173] [Q&A] [answered]
I have 15 minutes of audio data of a target speaker. I would like to understand which of the following would give the best results using espnet resources:
1. Fine-tuning e2e TTS(synthesizer+vocoder) to the target speaker data
2. Training a voice conversion model from an existing tts voice to target speaker voice(non-parallel data). During inference, generate the tts voice for the input text and convert that tts voice to target voice using vc model. 

Also, I could not find a proper way to do method-1 in this repo. Any help is very much appreciated. I also have only one target speaker. @kan-bayashi 

I was able to do method-1 using https://github.com/CorentinJ/Real-Time-Voice-Cloning (Tacotron2 + WaveRNN) but the results weren't always great.

**Comment by kan-bayashi** [comment, marked as answer]
For text2mel part,

1. First, make your own recipe https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/tts1/README.md#how-to-make-a-new-recipe-for-my-own-dataset
2. Then follow JVS recipe manner https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/tts1/README.md#how-to-finetune-the-pretrained-model

For vocoder part,
1. Make your own recipe https://github.com/kan-bayashi/ParallelWaveGAN/tree/master/egs#how-to-make-the-recipe-for-your-own-dateset
2. Run the recipe with `--pretrain` option.

I have not created vocoder adaptation example, but if I have a time, I will create example recipe like text2mel part.

**Reply by vishal16babu** [reply]
@kan-bayashi Thanks for pointing out the resources. Also, of the two approaches mentioned in the question, which one do you think would give the best results? I have tried the first approach using vcc20 baseline(TTS finetuned to target speaker) and the results(samples added below) are not that great. Do you think fine-tuning the vocoder for the target speaker will fix the issues?
[Original](https://t1856418.p.clickup-attachments.com/t1856418/85903da0-cb51-46d8-867d-b91facad1a4a/E10010.wav)
[Generated](https://t1856418.p.clickup-attachments.com/t1856418/818ff1bb-14ef-4886-ba24-9e12eceee72b/christopher_mid_E10010_gen.wav)
